test/tests/executioners/eigen_convergence/tests

```diff
@@ -0,0 +1,44 @@
+[Tests]
+  design = 'EigenProblem.md'
+  issues = '#20095'
+  [testb]
+    type = CSVDiff
+    csvdiff = 'b_out.csv'
+    input = 'b.i'
+    expect_out = '4 Nonlinear'
+    absent_out = '5 Nonlinear'
+    requirement = 'The system shall be able to solve an eigenvalue problem using something other than the L2 norm of Bx for normalization and a native eigenvalue solver.'
+    rel_err = 3e-5
+  []
+  [test11]
+    type = CSVDiff
+    csvdiff = 'a_out_eigen_0001.csv'
+    input = 'a.i'
+    expect_out = '1[01] Nonlinear'
+    absent_out = '12 Nonlinear'
+    cli_args = '-eps_tol 2e-10'
+    requirement = 'The system shall be able to solve an eigenvalue problem using the L2 norm of Bx and the sign of its first nonzero entry for normalization with a SLEPc eigenvalue solver.'
+  []
+  [test7]
+    type = CSVDiff
+    csvdiff = 'a_out_eigen_0001.csv'
+    prereq = test11
+    input = 'a.i'
+    expect_out = '7 Nonlinear'
+    absent_out = '8 Nonlinear'
+    cli_args = 'Problem/bx_norm=fluxintegral -eps_tol 2e-10'
+    petsc_version = '>=3.20.0'
+    requirement = 'The system shall be able to solve an eigenvalue problem using the sign of the first nonzero entry of Bx combined with something other than the L2 norm of Bx for normalization with a SLEPc eigenvalue solver.'
+  []
+  [test4]
+    type = CSVDiff
+    csvdiff = 'a_out_eigen_0001.csv'
+    prereq = test7
+    input = 'a.i'
+    expect_out = '4 Nonlinear'
+    absent_out = '5 Nonlinear'
+    cli_args = 'Problem/bx_norm=fluxintegral -eps_power_sign_normalization 0 -eps_tol 2e-10'
```

Comment:
Why is this test not taking 4 free power iterations?

Reply:
I remember when I tested this on a bigger Griffin problem, the convergence was better without `-eps_power_sign_normalization 0`. I think this may depend on the spectrum of the operator

Reply:
> Why is this test not taking 4 free power iterations? Oh, sorry, I did not add -eps_tol 2e-10. But I think this should happen automatically. 

This might require another change in SLEPc. I will see

Reply:
Ok I was able to force the number of free power iterations with just a MOOSE change